src/sbd-md.c

```diff
@@ -1046,11 +1046,6 @@ int servant(const char *diskname, int mode, const void* argp)
 	char uuid[37];
 	const struct servants_list_item *s = argp;
 
-	if (!diskname) {
-		cl_log(LOG_ERR, "Empty disk name %s.", diskname);
-		return -1;
-	}
-
```

Comment:
also unrelated to this PR, it would be a good idea to rename this function so it isn't overloaded by e.g. the argument to sbd_is_disk() etc.